Assets/Mapzen/TileAddress.cs

```diff
@@ -14,16 +14,15 @@ public TileAddress(int x, int y, int z)
 
         public static TileAddress FromLngLat(LngLat lngLat, int zoom)
         {
-            double tileResolution = Geo.EarthCircumferenceMeters / Geo.TilePixelSize;
-            double tileSize = tileResolution / (1 << zoom);
+            double tileSize = Geo.EarthCircumferenceMeters / (1 << zoom);
 
             MercatorMeters meters = Geo.Project(lngLat);
 
             double px = meters.x / tileSize;
             double py = meters.y / tileSize;
 
-            int tileX = (int)(Math.Ceiling(px / Geo.TilePixelSize) - 1);
-            int tileY = (int)(Math.Ceiling(py / Geo.TilePixelSize) - 1);
+            int tileX = (int)(Math.Ceiling(px) - 1);
```

Comment:
Yes, there is several options in order to perform the wrapping, one idea added in https://github.com/tangrams/tangram-unity/pull/35/commits/5aa8059aa4fa68b1b608cfb4f24eb1648dd9e2a7 is to leave the possibility to have negative tile ids (in order to compute offsets relative to other neighboring tiles) and only use tile wrapping to compute the tile origin, and when sending tile requests.